setup.py

```diff
@@ -25,15 +25,14 @@
     long_description=open('README.md').read(),
     description='A package for DigiCam pipeline',
     install_requires=[
-        'ctapipe>=0.6.0',
         'numpy',
         'matplotlib',
         'scipy',
         'astropy',
         'h5py',
         'tqdm',
     ],
-    tests_require=['pytest>=3.0.0'],
+    tests_require=['pytest>=3.0.0', 'ctapipe>=0.6.0'],
     setup_requires=['pytest-runner'],
```

Comment:
from https://setuptools.readthedocs.io/en/latest/setuptools.html#declaring-dependencies
it seems `install_requires` is the right place, `tests_require` is for package only needed during tests.

Reply:
Ok changing this